low-level/sensors/impl/generic/node_hw_alerts_info.py

```diff
@@ -135,5 +135,114 @@
                 "Power Supply regained the healthy state.", "None"
             )
         })
+    },
+    "Fan": {
+        **dict.fromkeys(["Fully Redundant", "Fully Redundant ()"], {
+            "Asserted": Alert(
+                "fault_resolved", "informational",
+                "'{} Sensor' has recovered from a failure. [{}]",
+                "None", "None"),
+            "Deasserted": Alert(
+                "fault", "critical",
+                "'{} Sensor', is Fully redundant . [{}]",
+                "Fan efficiency will be impacted",
+                DEFAULT_RECOMMENDATION)
```

Comment:
> "'{} Sensor', is Fully redundant . [{}]",

Description here looks doubtful, wrong

Since its fault, i believe it can be
Description -> {} sensor lost redundancy
Impact -> Other System Fans may run with increased speed to maintain optimal temperature
Recommendation -> Rectify Fan issue soon. If not sure, contact Seagate Support, as persistent problem may impact server availability

Reply:
In current code, the description is as 
Deasserted - Fully Redundant
alert_type = "fault"

does it Fan sensor lost redundancy?

Reply:
> Description -> {} sensor lost redundancy

{} sensor is running with full redundant speed. It might cause failure

Reply:
>{} sensor is running with full redundant speed. It might cause failure

what is full redundant speed. Its physical fan redundancy we are talking about

Reply:
addressed.